featuretools/io/sql.py

```diff
@@ -0,0 +1,107 @@
+import pandas as pd
+import sqlalchemy as sa
+
+import featuretools as ft
+from featuretools.utils.gen_utils import is_string
+
+
+def from_sql(id, connection=None, tables=None, index={},
```

Comment:
shouldn't use empty dictionaries as arguments to python functions

more info why here: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html